opentelemetry-api/src/opentelemetry/events/__init__.py

```diff
@@ -0,0 +1,61 @@
+from abc import ABC, abstractmethod
+from typing import Any, Optional
+
+from opentelemetry._logs import LogRecord
+from opentelemetry._logs.severity import SeverityNumber
+from opentelemetry.context.context import Context
+from opentelemetry.util.types import Attributes
+
+
+class Event(LogRecord):
+
+    def __init__(
+        self,
+        name: str,
+        payload: Optional[Any] = None,
```

Comment:
Any reason why we need `payload` and `attributes` when we can use the `LogRecord` `body` and `attributes` fields?

Reply:
I was trying to follow the semantic conventions: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/events.md where it said "In addition to a required name, an Event may contain a payload (data) of any type permitted by the [LogRecord body](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#field-body). In its implementation, the Event payload (data) will constitute the Body of the LogRecord. Like all other OpenTelemetry signals, an Event has optional attribute metadata that helps describe the event context.". Would you like me to rename `payload` as `body`?